sympy/integrals/integrals.py

```diff
@@ -394,6 +395,12 @@ def doit(self, **hints):
             return S.Zero
 
         # now compute and check the function
+        if isinstance(self.function, Sum):
+            for variable in self.variables:
+                if variable in self.function.limits[0]:
+                    break
+            else:
+                return Sum(Integral(self.function.function.doit(), self.variables[0]), self.function.limits).doit()
```

Comment:
perhaps the following?
```
_i =self
_sum = self.function
return _sum.func(_i.func(_sum.function, *_i.limits), *_sum.limits)
```

We are responding to integrate(Sum(...)) so I suspect that leaving the Sum part undone (and the interior as well) is more in line with what the user wrote. If they want the Sum computed they have to apply doit or they should have done `integrate(summation(...)...)` in the first place.